src/ucm/malloc/malloc_hook.c

```diff
@@ -17,7 +17,13 @@
 #include <ucm/api/ucm.h>
 #include <ucm/event/event.h>
 #include <ucm/mmap/mmap.h>
-#include <ucm/ptmalloc3/malloc-2.8.3.h>
+#if defined(HAVE_UCM_PTMALLOC283)
```

Comment:
why not #if HAVE_UCM_PTMALLOC283 / #ifdef HAVE_UCM_PTMALLOC283 ?
also - we might want to put these lines in separate file e.g ucm/malloc/allocator.h